client/components/phone-input/index.jsx

```diff
@@ -47,24 +56,33 @@ const PhoneInput = React.createClass( {
 		}
 	},
 
+	getFormattedNumberAndCountry( value ) {
+		let formattedNumber = value,
```

Comment:
The assignment of `formattedNumber = value` seems a little confusing here.
It seems that the only time formattedNumber remains `= value` is when `value` is falsey (it gets reassigned to `formatNumber(...)` otherwise).

Might something like this be clearer?

```es6
let formattedNumber;
let selectedCountry = this.state.selectedCountry;

if ( value ) {
    ...
    formattedNumber = formatNumber( value, selectedCountry );
}
```

Or do we need to pass the falsey value from `value`?